### ISSUE TITLE
OverFlowException

### ISSUE THREAD

**Issue by ZeekoZhu** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#43]
### Steps to reproduce
For example, there are 3 entities:
```
class Order
{
    public List<OrderItem> Items;
}

class OrderItem
{
    public Book Book;
}

class Book
{
    // An entity
}
```

unhandled exception occurred at:
```
DbContext.Orders.Include(o=>o.OrderItems).ThenInclude(oi=>oi.Book).AsNoTracking().SingleOrDefault(predicate);
```

### The issue

```
Exception message:"数组维度超过了支持的范围。"
Stack trace:
在 Pomelo.Data.MySql.Driver.GetColumns(Int64 count)
   在 Pomelo.Data.MySql.ResultSet.LoadColumns(Int64 numCols)
   在 Pomelo.Data.MySql.ResultSet..ctor(Driver d, Int32 statementId, Int64 numCols)
   在 Pomelo.Data.MySql.Driver.NextResult(Int32 statementId, Boolean force)
   在 Pomelo.Data.MySql.MySqlDataReader.NextResult()
   在 Pomelo.Data.MySql.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   在 Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.Execute(IRelationalConnection connection, String executeMethod, IReadOnlyDictionary`2 parameterValues, Boolean openConnection, Boolean closeConnection)
   在 Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.ExecuteReader(IRelationalConnection connection, IReadOnlyDictionary`2 parameterValues, Boolean manageConnection)
   在 Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable.Enumerator.MoveNext()
   在 Microsoft.EntityFrameworkCore.Query.QueryMethodProvider.<_ShapedQuery>d__3`1.MoveNext()
   在 Microsoft.EntityFrameworkCore.Query.QueryMethodProvider.<_Include>d__30`1.MoveNext()
   在 Microsoft.EntityFrameworkCore.Query.Internal.LinqOperatorProvider.ExceptionInterceptor`1.EnumeratorExceptionInterceptor.MoveNext()
   在 System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   在 BookList.Domain.Concrete.Repositories.OrderRepository.GetWithDetail(Func`2 predicate) 位置 C:\Users\vaezt\Source\Repos\BookList\BookList\src\BookList.Domain\Concrete\Repositories\OrderRepository.cs:行号 23
   在 BookList.ApiControllers.OrdersController.GetBookList(Int32 semesterId) 位置 C:\Users\vaezt\Source\Repos\BookList\BookList\src\BookList\ApiControllers\OrdersController.cs:行号 206
   在 lambda_method(Closure , Object , Object[] )
   在 Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeActionFilterAsync>d__28.MoveNext()
```

### Further technical details

MySQL version: 5.7
Operating system: Windows 10 14393.105
Pomelo.EntityFrameworkCore.MySql version: 1.0.0

Other details about my project setup:
project framework: 4.5.2

**Comment by Kagamine** [comment]
What is your default charset?

**Comment by ZeekoZhu** [comment]
utf8

**Comment by ZeekoZhu** [comment]
Now I find that this exception only occured when I query in a loop which interates a IQueryable list queried from DbContext,Once I convert the list to List<T> using ToList(), everything works well.

**Comment by Kagamine** [comment]
We've released 1.0.1-prerelease-10001, please try to use the latest version. If the issue still occur, please reopen this.

https://www.nuget.org/packages/Pomelo.EntityFrameworkCore.MySql/1.0.1-prerelease-10001
